app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -1300,13 +1305,26 @@ public void updateDisplay() {
             // Update the logo
             String imageUrl = ImageUtils.getLogoImageUrl(current, apiClient.getValue(), 440, false);
             if (imageUrl != null) {
-                Glide.with(requireContext())
-                        .load(imageUrl)
-                        .centerInside()
-                        .into(binding.itemLogo);
-                binding.itemLogo.setContentDescription(current.getName());
                 binding.itemLogo.setVisibility(View.VISIBLE);
                 binding.itemTitle.setVisibility(View.GONE);
+                Glide.with(requireContext())
+                    .load(imageUrl)
+                    .centerInside()
+                    .listener(new RequestListener<Drawable>() {
+                        @Override
+                        public boolean onLoadFailed(@Nullable GlideException e, Object model, Target<Drawable> target, boolean isFirstResource) {
+                            binding.itemLogo.setVisibility(View.GONE);
+                            binding.itemTitle.setVisibility(View.VISIBLE);
+                            return false;
+                        }
+
+                        @Override
+                        public boolean onResourceReady(Drawable resource, Object model, Target<Drawable> target, DataSource dataSource, boolean isFirstResource) {
```

Comment:
Put this method above the other one. All of our callback functions are in order of success,failed.